### PR TITLE
Check status for new element

### PR DESCRIPTION
**Pull request by Sixtinebs**
I create a fonction around add code who generate the checkItem of click and add this fonction when add new element

### CODE REVIEW

**Comment by tchapuis** [comment]
Does this works for already present elements ?

**Comment by Sixtinebs** [comment]
Oups I change that

File: src/assets/js/list.js [hunk 1]
```diff
@@ -7,6 +7,19 @@ const generateItem = (id, content) => {
   return newListElement;
 };
 
+const checkItem = (e) => {
+  const { itemId } = e.currentTarget.dataset;
+  document.querySelector(`#item-${itemId} > label`).classList.add('line-through');
```
Comment (tchapuis) [inline src/assets/js/list.js:12]: Can you use the `toggle` function instead of `add` please.

Comment (Sixtinebs) [reply src/assets/js/list.js:12]: yep :)

**Pull request merged**